includes/theme-compat/register-profile.php

```diff
@@ -0,0 +1,104 @@
+            <div class="form-box box-shadow">
+                <div class="section-heading">
+                    <?php if (!empty($page_name) && $page_name == 'profile') { ?> <h2 class="heading-title">My
+                        Profile</h2> <?php } ?>
+                    <?php if (!empty($page_name) && $page_name == 'register') { ?> <h2 class="heading-title">
+                        Register</h2> <?php } ?>
+                </div>
+                <form action="/form.php" id="register" method="post" enctype="multipart/form-data">
+                    <div class="flex-container justified-horizontally">
+                        <div class="primary-container">
+                            <h4 class="form-title">About me</h4>
+                            <input type="hidden" name="form_name" value="register"/>
+                            <div class="form-field-wrapper">
+                                <input type="text" name="first_name" placeholder="First Name*"
+                                       <?php if (!empty($_SESSION['first_name'])) {
+                                           echo 'value="' . $_SESSION['first_name'] . '" style="color: #3c71fe;" ';
+                                       } ?>required>
+                            </div>
+                            <div class="form-field-wrapper">
+                                <input type="text" name="last_name"
+                                       placeholder="Last Name*" <?php if (!empty($_SESSION['last_name'])) {
+                                    echo 'value="' . $_SESSION['last_name'] . '" style="color: #3c71fe;" ';
+                                } ?> required>
+                            </div>
+                            <div class="form-field-wrapper">
+                                <input type="email" name="email"
+                                       placeholder="Email*" <?php if (!empty($_SESSION['email'])) {
+                                    echo 'value="' . $_SESSION['email'] . '" style="color: #3c71fe;" ';
+                                } ?> required>
+                            </div>
+                            <?php if (!empty($page_name) && $page_name == 'register') { ?>
+                                <div class="form-field-wrapper">
+                                    <input type="password" name="password" placeholder="Password*" required>
+                                </div><?php } ?>
+                            <?php if (!empty($page_name) && $page_name == 'register') { ?>
+                                <div class="form-field-wrapper">
+                                    <input type="password" name="r_password" placeholder="Repeat Password*" required>
+                                </div><?php } ?>
+                            <?php if (!empty($page_name) && $page_name == 'profile') { ?>
+                                <div class="form-field-wrapper">
+                                    <input type="password"
```

Comment:
a bit weird formatting

Reply:
I made some formatting changes, but I'm not sure it has any effect on better readability.